glean-core/android/src/main/java/mozilla/telemetry/glean/testing/GleanTestRule.kt

```diff
@@ -28,8 +28,11 @@ import org.junit.runner.Description
  */
 @VisibleForTesting(otherwise = VisibleForTesting.NONE)
 class GleanTestRule(
-    val context: Context
+    internal val context: Context
```

Comment:
nit: maybe this can even just be private?

Reply:
hmmm, too many visibility options. What's the difference?

Reply:
ah, file-private (`private`) vs. module-private (`internal`). Changing it.